commons/src/main/java/com/simpletools/calculator/commons/operations/InvOperation.kt

```diff
@@ -0,0 +1,12 @@
+package com.simpletools.calculator.commons.operations
+
+import com.simpletools.calculator.commons.operations.base.UnaryBitwiseOperation
+
+object InvOperation : UnaryBitwiseOperation() {
+
+    override fun getResult() = value.inv()
+
+    override fun applyOperator(str: String): String {
+        return "~($str)"
```

Comment:
What does the ~ signify ?

Reply:
inv operation, so formula for the inv of 5 is ~(5)